pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -291,7 +291,8 @@ func (l Logger) Printf(format string, args ...interface{}) {
 }
 
 func init() {
-	metrics.RegisterMetrics()
+	metrics.RegisterAPIMetrics()
```

Comment:
one suggestion here, instead of having 2 calls here, how about we define RegisterMetrics() inside metrics.go , which calls all relevant metrics, so that if any new metrics added , we need to change only in metrics code
and here we can have a generic call metrics.RegisterMetrics()